tests/light/conftest.py

```diff
@@ -114,6 +114,24 @@ def syslog_ng(request, testcase_parameters):
     return syslog_ng
 
 
+class FdCloseCallback:
```

Comment:
I would use a bit more generic name here. This is actually a testcase teardown function, isn't it?

Is it limited to doing the closure of FDs? In my understanding this could be any cleanup function, closing an FD being a prime example.

Reply:
Thanks for the naming suggestion. 

Yes, I would not pass `request` as a parameter because of the following:
 - it always needs some clarification about the naming: what is it, where does it came from and so on
 - and it's functionality is too broad: it could do a lot of things
 - and with the `request` for the new pytest users it is hard to understand how the teardown works for the testcases (`yield` or `request.addfinalizer`)

I think it is better to use a parameter with the name of `teardown` itself instead of `request` or what I currently uses in the PR `fd_close_callback`. Currently I have checked that the `teardown` name is not a reserved keyword so we could use it as a fixture name.

Reply:
In my last push I have added the new rename concept.